runtime/lua/vim/lsp/completion.lua

```diff
@@ -403,6 +403,10 @@ local function trigger(bufnr, clients)
   reset_timer()
   Context:cancel_pending()
 
+  if tonumber(vim.fn.pumvisible()) == 1 and Context.isIncomplete then
```

Comment:
Ah, just realized I was too quick with the merge. This should be `not Context.isIncomplete`, no?
If the previous result was incomplete it must fire another request to fetch more candidates.

Reply:
i mixedup the var name with boolean value T.T